test/network_proposal_test.rb

```diff
@@ -20,11 +20,13 @@
   end
 
   describe "#make_proposal" do
+    let(:settings) { Y2Network::ProposalSettings.create_instance }
     let(:using_wicked) { true }
```

Comment:
Why not call it `use_network_manager`? You're using it to mock the return value of `Yat::Lan::UseNetworkManager`.